### PR TITLE
Update extension compatibility for 2.4.0

### PR DESCRIPTION
**Pull request by jeff-matthews**
## Purpose of this pull request

This pull request (PR) updates Magento-developed extension (MDE) and vendor-bundled extension (VBE) compatibility for 2.4.0.

## Affected DevDocs pages

- https://devdocs.magento.com/release/#compatibility
- https://devdocs.magento.com/extensions/vendor/#compatibility

whatsnew
Updated [Magento-developed extension (MDE)](https://devdocs.magento.com/release/#compatibility) and [vendor-bundled extension (VBE)](https://devdocs.magento.com/extensions/vendor/#compatibility) compatibility for Magento 2.4.0.

### CODE REVIEW

File: src/_data/mde.yml [hunk 1]
```diff
@@ -17,7 +18,8 @@ extensions:
           2.3.2: supported
           2.3.3: supported
           2.3.4: supported
-          2.3.5-p1: supported 
+          2.3.5-p1: supported
+          2.4.0: supported
```
Comment (hickskenh) [inline src/_data/mde.yml:22]:
```suggestion
          2.4.0: not supported
```

File: src/_data/mde.yml [hunk 2]
```diff
@@ -26,7 +28,8 @@ extensions:
           2.3.2: compatible
           2.3.3: compatible
           2.3.4: compatible
-          2.3.5-p1: not supported     
+          2.3.5-p1: not supported
+          2.4.0: compatible
```
Comment (hickskenh) [inline src/_data/mde.yml:32]:
```suggestion
          2.4.0: not supported
```

**Comment by jeff-matthews** [comment]
Thanks @hickskenh!

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**